src/main/java/net/sf/jabref/external/push/PushToApplicationButton.java

```diff
@@ -56,9 +54,10 @@
     private final JPopupMenu optPopup = new JPopupMenu();
     private final JMenuItem settings = new JMenuItem(Localization.lang("Settings"));
 
-    public PushToApplicationButton(JabRefFrame frame, List<PushToApplication> pushActions) {
-        this.frame = frame;
-        this.pushActions = pushActions;
+
+    public PushToApplicationButton(JabRefFrame frameJR, List<PushToApplication> pushActionsList) {
+        frame = frameJR;
+        pushActions = pushActionsList;
```

Comment:
Yeah, have seen this in @oscargus' warning-removal-PRs. 
I don't have a strong opinion on this: I don't bother if a member name is reused in a method (especially not in a constructor/setter)...